index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <section id="suzhouhe">
+        <h1>
+          Suzhou River
+        </h1>
+        <img
+          src="https://cdn.glitch.com/cb877aac-d372-492e-b711-f42985913fe5%2F%E4%BC%98%E5%8C%96%E5%90%8E.jpg?v=1602752962003
+      "
+          alt="film poster"
+        />
+        <h3>
+          2000 | romance | 83min | 7.4
+          <a href="https://www.youtube.com/watch?v=3GLEiu-VhvE">
+            <br />
+            watch trailer</a
+          >
+        </h3>
```

Comment:
A heading is like a title of a thing -- a page, a section, etc. While this is all important info, it should either be:
- separate heading elements, or
- wrapped in another HTML element (maybe a list, since these are all attributes of the film?)

But they should not, however, be all bundled up as a single heading element.

There is a reading for the current week about [HTML lists](https://mica-web.github.io/learn/html/lists).